addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18681,7 +18681,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36223"
-msgid "By default, the guide data is cached in a local database to speed up importing on startup."
+msgid "Activate to store epg data in a database. This may speedup importing epg data on startup."
```

Comment:
And in case you change the label above, we could write something like this here:
`Activate to cache the epg data of remote PVR connections. Depending on the connection, this may speed up the availability of epg data on startup.`

Reply:
I don't want to change the label text.